clients/go/tests/integration/createJob_test.go

```diff
@@ -0,0 +1,144 @@
+package integration
+
+import (
+    "encoding/json"
+    "fmt"
+    . "github.com/onsi/ginkgo"
+    . "github.com/onsi/gomega"
+    "github.com/zeebe-io/zeebe/clients/go"
+    "github.com/zeebe-io/zeebe/clients/go/commands"
+)
+
+type MyJob struct {
+    ID string
+    Name string
+}
+
+func (job MyJob) String() string {
+    return fmt.Sprintf("{\"name\": \"%s\"}", "myjob")
```

Comment:
I would like to also have the ID field in the string to make it consistent with the struct above.

Reply:
Why does it matter? The idea behind this is that user can construct custom JSON which does not have to be related to the field of the struct. To make this idea more clear - that's why this test is expressed like this.